src/main/resources/default-workflows/Machine_Learning/DLM_AnoDet_a_NS.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.8"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.8 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.8/schedulerjob.xsd"
+    name="DLM_AnoDet_a_NS" 
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2"
>
```

Comment:
Strange, no idea what is it. When I export it from Studio, it seems to be fine, but when uploaded in github, there are two more characters.

![image](https://user-images.githubusercontent.com/9742161/27291238-f6126eae-550f-11e7-8a12-f042dee9828d.png)